mcux/CMakeLists.txt

```diff
@@ -56,3 +56,4 @@ add_subdirectory_ifdef(
 enable_language(C ASM)
 
 zephyr_sources_ifdef(CONFIG_SOC_LPC54114_M4 devices/${MCUX_DEVICE}/gcc/startup_LPC54114_cm4.S)
+zephyr_sources_ifdef(CONFIG_SOC_MIMX8MM6 devices/${MCUX_DEVICE}/system_MIMX8MM6_cm4.c)
```

Comment:
We haven't been using this file for other nxp socs because it's mostly redundant with zephyr startup. Is there a particular reason you need it for this soc?

Reply:
It has the definition for `SystemCoreClockUpdate` function used in `soc.c` file.

Reply:
We don't need `SystemCoreClockUpdate` because there is nothing consuming the variable `SystemCoreClock`